src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -150,6 +226,22 @@
     )
     protected Transform transform = Transform.RAW;
 
+    @Argument(
+            doc = "File containing the exons for analysis",
+            shortName = EXOME_FILE_SHORT_NAME,
+            fullName = EXOME_FILE_FULL_NAME,
+            optional = true
+    )
+    protected File exomeFile = null;
```

Comment:
Did you consider making this a `FeatureInput<BEDFeature>` rather than a `File` so that the engine would take care of initializing the codec, reader, etc. for the BED file, and provide access the exons overlapping each read via the `FeatureContext` in `apply()`? Or would this not be helpful for your use case?